Bio/SeqIO/InsdcIO.py

```diff
@@ -376,6 +380,19 @@ class _InsdcWriter(SequenceWriter):
     )
 
     def _write_feature_qualifier(self, key, value=None, quote=None):
+        if not all(ch in _allowed_table_component_name_chars for ch in key):
```

Comment:
Replace that with a set operation: ``_allowed_table_component_name_chars.issuperset(key)``

Reply:
Cool, I hadn't thought of `. issuperset`. I just check it and it is faster than `all()`. I'll add another commit in a second.